### PR TITLE
Move oVirt settings to the oVirt provider repository

### PR DESCRIPTION
**Pull request by borod108**
These settings will be removed from the core repo in a separate PR.
The settings in the provider repo override the once in the core repo
in case a setting is defined in both.

### CODE REVIEW

**Comment by borod108** [comment]
@miq-bot assign @masayag

**Comment by borod108** [comment]
@masayag please review

**Comment by borod108** [comment]
@miq-bot add-label wip

**Comment by borod108** [comment]
@miq-bot remove-label wip

**Comment by borod108** [comment]
@miq-bot assign @jhernand 
@jhernand Hi, are the known issues regarding v4 provisioning fixed? if so, can you please review this PR?

**Comment by borod108** [comment]
@masayag Juans fix was merged, can you please review?
@miq-bot assign @masayag

File: config/settings.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ---
 :ems:
+  :ems_redhat:
+      :use_ovirt_engine_sdk: true
```
Comment (masayag) [inline config/settings.yml]: shouldn't setting the default value to 'true' be introduced in a separate PR ?
